File: wrappers/mvc/demos/Kendo.Mvc.Examples/Models/Map/MapDataRepository.cs
using System.Collections.Generic;
using System;

namespace Kendo.Mvc.Examples.Models
{
    public partial class MapDataRepository
    {
        public static IList<Marker> StoreLocations() 
        {
            return new Marker[]
            {
                new Marker(30.2675,-97.7409, "Zevo Toys"),
                new Marker(30.2707,-97.7490, "Foo Bars"),
                new Marker(30.2705,-97.7409, "Mainway Toys"),
                new Marker(30.2686,-97.7494, "Acme Toys")
            };
        }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/models/MapDataRepository.java
package com.kendoui.spring.models;

import java.util.ArrayList;
import java.util.List;

public class MapDataRepository {
    public static List<Marker> StoreLocations() 
    {
        List<Marker> result = new ArrayList<Marker>();
        result.add(new Marker(30.2675,-97.7409, "Zevo Toys"));
        result.add(new Marker(30.2707,-97.7490, "Foo Bars"));
        result.add(new Marker(30.2705,-97.7409, "Mainway Toys"));
        result.add(new Marker(30.2686,-97.7494, "Acme Toys"));
        
        return result;
    }
}
